apps/api/src/bootstrap.ts

```diff
@@ -88,8 +91,20 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   app.use(extendedBodySizeRoutes, bodyParser.json({ limit: '20mb' }));
   app.use(extendedBodySizeRoutes, bodyParser.urlencoded({ limit: '20mb', extended: true }));
 
-  app.use(bodyParser.json());
-  app.use(bodyParser.urlencoded({ extended: true }));
+  let rawBodyBuffer: undefined | (() => void) = undefined;
+
+  try {
+    if (process.env.NOVU_ENTERPRISE === 'true' || process.env.CI_EE_TEST === 'true') {
```

Comment:
I see that we are using this OR condition in several places, but the `CI_EE_TEST` flag only ever occurs next to the `NOVU_ENTERPRISE` flag. This extra flag seems redundant and adds [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity).

Can we simplify to the following?
```suggestion
    if (process.env.NOVU_ENTERPRISE === 'true') {
```